__tests__/lint-config/invalid-definition-and-config/snapshot.js

```diff
@@ -17,82 +17,6 @@ Error was generated by the configuration spec rule.
 
 
 ❌ Your config has 1 error.
```

Comment:
Do we need to cover this also with the `warn` severity to test the openapi errors?